contrib/server/operations/pbench-upload-results.py

```diff
@@ -52,32 +51,27 @@ def get_md5(tarball: Path) -> str:
 
 def upload(
     server: str, token: str, tarball: Path, metadata: Optional[list[str]] = None
-) -> Response:
-    query_parameters = {}
-
+) -> requests.Response:
     md5 = get_md5(tarball)
-    query_parameters["access"] = "public"
-    dataset = tarball.name
-    satellite, _ = tarball.parent.name.split("::")
-    meta = [
-        f"global.server.legacy.migrated:{datetime.datetime.now(tz=datetime.timezone.utc):%Y-%m-%dT%H:%M}"
-    ]
-    if satellite:
+    uploaded = datetime.datetime.fromtimestamp(
+        tarball.stat().st_mtime, tz=datetime.timezone.utc
+    )
+    meta = [f"global.server.legacy.migrated:{uploaded:%Y-%m-%dT%H:%M}"]
```

Comment:
I sure hope we never care about the "seconds" value....

Reply:
In testing, I have narrowed the range to seconds to get a reasonable number of tarballs; in real use I can't imagine we'd ever care and I decided to keep it compact.

Reply:
I'm sure that you're right, but I'm just thinking about us trying to differentiate between two tarballs, and having "more uniqueness" is usually handy, but nevermind.